lang/java/reef-tang/tang/src/main/java/org/apache/reef/tang/formats/ConfigurationModule.java

```diff
@@ -47,6 +49,10 @@
  * See org.apache.reef.tang.formats.TestConfigurationModule for more information and examples.
  */
 public class ConfigurationModule {
+
+  private static final String CLASS_NAME = ConfigurationModule.class.getName();
```

Comment:
Why the added field for the `CLASS_NAME`? Can't we just inline this call in the `getLogger()` call?